libs/@blockprotocol/graph/src/graph-block-handler.ts

```diff
@@ -41,7 +41,13 @@ export class GraphBlockHandler<Temporal extends boolean>
   implements
     Omit<
       BlockGraphMessages<Temporal>,
-      "createEntityType" | "updateEntityType" | "deleteEntityType"
+      | "createEntityType"
+      | "updateEntityType"
+      | "deleteEntityType"
```

Comment:
Should we just remove this method?

Reply:
Yeah – done in https://github.com/blockprotocol/blockprotocol/pull/946/commits/49e085829cbe652b1b8cae115a451722b6aeda63